packages/browser-wallet/src/popup/pages/AgeProofRequest/i18n/en.ts

```diff
@@ -0,0 +1,11 @@
+const t = {
+    moreDetails: 'More details',
+    verify: 'Verify',
+    ageMin: 'Verify that you are older than {{ age }}',
+    ageMax: 'Verify that you are younger than {{ age }}',
```

Comment:
Fix the texts, e.g. Verify that you are {{ age }} or younger.

Reply:
Changed it to conform to the way we write in the main proofs.